### ISSUE TITLE
Not compiling properly, saying something wrong in concatenation method

### ISSUE THREAD

**Issue by sanjabon** [Grzego/handwriting-generation#21]

**Comment by Grzego** [comment]
@sanjabon Please provide more details (like console error).

**Comment by sanjabon** [comment]
![image](https://user-images.githubusercontent.com/60978828/82743915-6fcfe080-9d8f-11ea-98ee-7f761b93d819.png)
I am beginner in running files maybe.
From the description you provided, i run "python preprocess.py" but python doesn't mean anything in says terminal. So i executed same file using python 3 and got this error.

**Comment by Grzego** [comment]
Did you modified the file? There should be no `[]` in this line, look [L109](https://github.com/Grzego/handwriting-generation/blob/master/preprocess.py#L109). Try restoring the original file (or just remove `[]`. if that's all). :)

**Comment by sanjabon** [comment]
Hey, Thank you. But I didn't modified the file i think. How can i make cool stuff like this?

**Comment by sanjabon** [comment]
![image](https://user-images.githubusercontent.com/60978828/82744721-ff7a8c80-9d99-11ea-89b7-d14f23e3b0ed.png)
Now getting this error while executing train.py
i googled the issue, your project was done using old tensorflow version, now giving syntax errors.

**Comment by Grzego** [comment]
Yes, it looks like error from TensorFlow 2.0. You can try installing older versions (if possible), for example 1.15 should work fine.
